core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/PricingPhase.ios.kt

```diff
@@ -1,9 +1,18 @@
 package com.revenuecat.purchases.kmp.models
 
-public actual data class PricingPhase (
-    public actual val billingPeriod: Period,
-    public actual val recurrenceMode: RecurrenceMode,
-    public actual val billingCycleCount: Int?,
-    public actual val price: Price,
-    public actual val offerPaymentMode: OfferPaymentMode?,
-)
+public actual class PricingPhase
+
+public actual val PricingPhase.billingPeriod: Period
+    get() = error("Not available in iOS")
```

Comment:
Is this ok? Devs should never get an instance of this object in iOS right now... So I thought this was the simplest.

Reply:
Yeah I think that's fine. This is also temporary right?

Reply:
Yup, once we move away from `expect`/`actual` this shouldn't be an issue.